modules/builtin/src/backend/prompts/common.ts

```diff
@@ -0,0 +1,18 @@
+import { FormField } from 'common/typings'
+
+const commonFields = (defaultDuration: number = 5, defaultSearchBackCount: number = 0): FormField[] => {
+  return [
+    {
+      type: 'text',
+      key: 'duration',
+      label: 'module.builtin.duration'
+    },
+    {
+      type: 'text',
+      key: 'searchBackCount',
+      label: 'module.builtin.searchBackCount'
+    }
+  ]
+}
```

Comment:
The default values aren't used for now I haven't found a way to set them with content forms @MaxCloutier. Also I think a number field type with front-end validation would be useful.